src/components/StepperVertRight/index.js

```diff
@@ -18,44 +14,17 @@ const POS_BETTING = 1;
 const POS_ORACLE_RESULT_SETTING = 2;
 const POS_OPEN_RESULT_SETTING = 3;
 
-const messages = defineMessages({
-  block: {
```

Comment:
It's better to keep this. otherwise the script cannot catch the localization string.

Reply:
Are you sure? I just read that all it does is get the language object. [see here](https://github.com/yahoo/react-intl/wiki/API#definemessages)

Reply:
I mean 'npm run build:langs', this cannot extract the strings to language files.

Reply:
why don't we come back to this later so we can keep moving

Reply:
created an issue. You can keep moving. You can fix it after merge.